src/main/java/io/micronaut/gradle/docker/MicronautDockerfile.java

```diff
@@ -30,8 +30,7 @@
     private final Property<DockerBuildStrategy> buildStrategy;
     @Input
     private final Property<String> defaultCommand;
-    @Input
-    private final ListProperty<String> customEntrypoint;
+
```

Comment:
Sorry that's not what I meant. I meant that the _property name_ was wrong: it should be `entryPoint`, and its default value should be configured with the `getArgs().map...` thing.

Reply:
@melix with the current approach as it exists we are able to leverage the existing `entryPoint` method that exists in the base `Dockerfile` task. I worried about overloading the method with a property of the same name that does essentially the same thing being confusing and not clear to a user.

In fact my first approach to solving #237 was to use the base dockerfile `entryPoint` method, it just generated the resulting Dockerfile incorrectly because that task doesn't have any logic to limit you to one entrypoint.

Reply:
No worries, let me take a look at this.